Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,24 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+//Answer :
+function getAngleType (angle) {
```

Comment:
There are still some cases that are not covered. Not all cases are explicitly mentioned in the specifications ;)

Reply:
nice. Thanks